strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/main/java/org/carlspring/strongbox/config/NpmLayoutProviderConfig.java

```diff
@@ -42,7 +43,12 @@
     @Bean
     public ObjectMapper npmJackasonMapper()
```

Comment:
There's actually a typo here -- it's supposed to be `npmJacksonMapper`, not `npmJackasonMapper`. (Too many `a`-s in `Jackson`).

Reply:
@sbespalov : I am merging this pull, but could you please fix the typo in a follow-up pull? Thanks! :)

Reply:
https://github.com/strongbox/strongbox/pull/792